packages/example-vue/src/mocks/handler.ts

```diff
@@ -1,11 +1,16 @@
-import { rest } from 'msw';
+import { http, delay, HttpResponse } from 'msw';
 
 export const testHandlers = [
-  rest.get('/api/users', async (_, response, context) => {
-    return response(
-      context.status(200),
-      context.delay(500),
-      context.json([{ name: 'Harry' }, { name: 'Ron' }, { name: 'Hermione' }])
-    );
-  }),
+  http.get(
+    '/api/users',
+
+    async () => {
+      await delay(500);
```

Comment:
Wouldn't it be nice(r) the use the default `delay()` which results in random (realistic) delays?

https://mswjs.io/docs/api/delay/#implicit-delay

```ts
// Await a random realistic server response time.
await delay()
```

fostar82
--


<br class="Apple-interchange-newline">

Reply:
Agreed, a random delay would be a lot nicer!